@here/harp-examples/src/datasource_features_polygons.ts

```diff
@@ -282,7 +282,7 @@ export namespace PolygonsFeaturesExample {
             canvas,
             theme: "resources/berlin_tilezen_night_reduced.json",
             projection: sphereProjection,
-            clipPlanesEvaluator: new AltitudeBasedClipPlanesEvaluator(300000)
+            clipPlanesEvaluator: new TopViewClipPlanesEvaluator(300000)
```

Comment:
This should be `TiltViewClipPlanesEvaluator` since @Astrak will merge Globe gestures soon.